examples/md-flexible/src/Simulation.cpp

```diff
@@ -178,6 +178,22 @@ void Simulation::finalize() {
   logMeasurements();
 }
 
+// Note: An initial force calculation should actually take place before the first position update according to
+// Störmer Verlet the literature (Griebel et al., Numerical simulation in molecular dynamics: numerics, algorithms,
+// parallelization, applications, 2007). In this implementation, the first position update is calculated with zero
+// forces and only the velocity is used. This means that essentially, the molecules do not start the simulation at
+// the given positions but are already moved by one timestep in the direction of their initial velocity. If you want
+// to observe the exact movement of particles, you should bear this in mind.
+// The initial force calculation is omitted here for the following reasons:
+// 1. md-flexible is a test application to demonstrate how AutoPas can be used and not a software for physically
+// correct simulation.
+// 2. md-flexible is often used during development for performance measurements or memory analysis. An additional
+// force calculation step would unnecessarily complicate the simulation loop and distort the statistics.
+// 3. We also do not need an initial force calculation or the oldForce for checkpoint loading. Reason: In the last
+// simulation step before saving the checkpoint, F was calculated and a velocity update was performed with this F. In
+// the first iteration after loading the checkpoint file, the position update is calculated with this F (loaded from the
+// checkpoint file) from the last simulation step, then F_old is set to F and the new force is calculated. This
+// automatically continues the Störmer Verlet algorithm seamlessly.
```

Comment:
```suggestion
// Note: An initial force calculation should actually take place before the first position update, according to
// Störmer Verlet the literature (Griebel et al., Numerical simulation in molecular dynamics: numerics, algorithms,
// parallelization, applications, 2007). In this implementation, the first position update is calculated with zero
// forces, if none are given initially, and only the velocity is used. This means that, essentially, the molecules do not start the simulation at
// the given positions but are already moved by one timestep in the direction of their initial velocity. If you want
// to observe the exact movement of particles, you should bear this in mind.
// The initial force calculation is omitted here for the following reasons:
// 1. The primary focus for md-flexible is to be a test application and demonstrator of how AutoPas can be used. Not to be a perfect simulator.
// 2. md-flexible is often used during development for performance measurements or memory analysis. An additional
// force calculation step would unnecessarily complicate the simulation loop and distort the statistics.
// 3. We also do not need an initial force calculation or the oldForce for checkpoint loading. Reason: In the last
// simulation step before saving the checkpoint, F was calculated, and a velocity update was performed with this F. In
// the first iteration after loading the checkpoint file, the position update is calculated with this F (loaded from the
// checkpoint file) from the last simulation step, then F_old is set to F and the new force is calculated. This
// automatically continues the Störmer Verlet algorithm seamlessly.
```

(might need reformatting after my suggestions)

I'd prefer if this were moved to the doxygen doc block in `Simulation.h` as a `@note`. Please make sure that the whole block is then also rendered as a note and not only the first sentence by building it locally.

Reply:
Do you want the comment directly above the `class Simulation` as note, or in the doxygen doc of `void run()`?

Reply:
Good question. I'd put it in the doc for `run()`